### PR TITLE
DE47953: LOR/LTI/SCORM > Emebedded preview can resize horizontally

### PR DESCRIPTION
**Pull request by coffeexcode**
## Relevant Rally
- [DE47953](https://rally1.rallydev.com/#/?detail=/defect/630637740037&fdp=true): LOR/LTI/SCORM > Emebedded preview can resize horizontally

## Description
Originally I had updated the previews to `resize: both` with some CSS to make the draggable handle work better. I had missed that it should only be `resize: vertical` so I have cleaned up some of the CSS to make the preview containers look a little bit better.

Unfortunately, the way to fullscreen view works on different content items is a little weird. If you try to make the iframe resize, it's basically impossible to grab the resize handle (on Firefox sometimes doesn't even show up). So to make it easy to grab and resize, I applied this to the containing div and moved it below the framed content. This way it works for all the fullscreen views as well as LTI.

### CODE REVIEW

File: components/d2l-activity-editor/d2l-activity-content-editor/shared-components/d2l-activity-content-link-preview.js [hunk 1]
```diff
@@ -71,8 +61,8 @@ class ContentEditorLinkPreview extends SkeletonMixin(LocalizeActivityEditorMixin
 		}
 
 		return html`
-			<div class="d2l-skeletize d2l-resizable">
-				<div class="d2l-link-preview-iframe-container">
+			<div class='d2l-skeletize'>
```
Comment (coffeexcode) [inline components/d2l-activity-editor/d2l-activity-content-editor/shared-components/d2l-activity-content-link-preview.js:64]: I accidentally changed this to double quotes in my previous PR. They should be singles

File: components/d2l-activity-editor/d2l-activity-content-editor/imported-scorm/d2l-activity-content-imported-scorm-activity-detail.js [hunk 2]
```diff
@@ -51,6 +51,9 @@ class ContentImportedScormActivityDetail extends AsyncContainerMixin(SkeletonMix
 					margin-left: 40px;
 					margin-right: 0;
 				}
+				d2l-activity-content-imported-scorm-preview {
+					padding-bottom: 18px;
```
Comment (brissons) [inline components/d2l-activity-editor/d2l-activity-content-editor/imported-scorm/d2l-activity-content-imported-scorm-activity-detail.js:55]: nice! much needed here

File: components/d2l-activity-editor/d2l-activity-content-editor/shared-components/d2l-activity-content-link-preview.js [hunk 3]
```diff
@@ -31,19 +31,9 @@ class ContentEditorLinkPreview extends SkeletonMixin(LocalizeActivityEditorMixin
 					iframe height should be 60% of viewport height,
```
Comment (brissons) [inline components/d2l-activity-editor/d2l-activity-content-editor/shared-components/d2l-activity-content-link-preview.js:31]: does this comment still apply? is the behavior different?

File: components/d2l-activity-editor/d2l-activity-content-editor/shared-components/d2l-activity-content-link-preview.js [hunk 4]
```diff
@@ -31,19 +31,9 @@ class ContentEditorLinkPreview extends SkeletonMixin(LocalizeActivityEditorMixin
...
-				}
-				.d2l-resizable {
-					min-height: 60vh;
-					overflow: auto;
-					padding: 0 12px 12px 0;
-					resize: both;
-				}
-				:host([dir="rtl"]) .d2l-resizable {
-					padding: 0 0 12px 12px;
+					resize: vertical;
```
Comment (brissons) [inline components/d2l-activity-editor/d2l-activity-content-editor/shared-components/d2l-activity-content-link-preview.js:36]: do we want a minimum height? it looks like the user might now be able to resize it to be `0px`

**Review by Kiefer-Co-D2L — APPROVED** [review]
No More Horizontal Resize

Gave this one a dev test: the horizontal resizing is now gone, hooray!

**Review by kristian-torres — APPROVED** [review]

**Pull request merged**